packages/actions/index.ts

```diff
@@ -463,8 +463,13 @@ export const getFriendRequests = () => {
     const user = getUser(getState())()
     const rawPendingFriends = await creditProtocol.getFriendRequests(user.address)
     const pendingFriends = rawPendingFriends.map(jsonToPendingFriend)
+
+    const rawPendingOutboundFriends = await creditProtocol.getOutboundFriendRequests(user.address)
+    const pendingOutboundFriends = rawPendingOutboundFriends.map(jsonToPendingFriend)
+
+    console.log('THIS IS IT ', pendingFriends, pendingOutboundFriends)
```

Comment:
Can we add keys like this to 2 rows in `friends/index.ts` ? I'm getting React Native warnings.
<img width="641" alt="image" src="https://user-images.githubusercontent.com/1081476/44234311-2c2c5980-a15b-11e8-8b71-1562b28fa085.png">